crates/compiler/gen_llvm/src/llvm/lowlevel.rs

```diff
@@ -1172,9 +1173,68 @@ pub(crate) fn run_low_level<'a, 'ctx>(
             // which could be useful to look at when implementing this.
             todo!("implement checked float conversion");
         }
-        I128OfDec => {
-            arguments!(dec);
-            dec_unary_op(env, bitcode::DEC_TO_I128, dec)
+        NumWithoutDecimalPoint | NumWithDecimalPoint => {
+            arguments!(arg);
+            arg
+        }
```

Comment:
Is this correct?

Reply:
It work's for me...
input is an I128, output is the same I128, so that was the most simple solution for me. But I only tested it on my linux machine.

Reply:
:+1: I forgot `Dec` was an `I128` under the hood :smile: